tests/typerel/trectype.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   errormsg: "internal error: cannot generate C type for: PA"
-  disabled: true
+    disabled: true
```

Comment:
Disabled around 2016-12.
This seems to correctly validate errors in recursive types, just that the message changed over time.
Running it gives:
	
			Expected:
			internal error: cannot generate C type for: PA

			Gotten:
			illegal recursion in type 'PA'
			
seems fixable

Reply:
Yeah, let's migrate to spec.